rust_src/src/casefiddle.rs

```diff
@@ -7,13 +7,37 @@ use keymap::Ctl;
 use lisp::defsubr;
 use lisp::LispObject;
 use lists::put;
-use remacs_sys::{case_action, casify_object, casify_region_nil, casify_word};
+use remacs_sys::{case_action, casify_object, casify_region_nil};
+use remacs_sys::{casify_region, scan_words, set_point};
```

Comment:
We try to keep the import order consistent. You can take a look at the order in the other files. I think all non lisp functions should be in one import.